WordPress/Classes/ViewRelated/Blog/SiteSettingsViewController+Swift.swift

```diff
@@ -100,25 +100,34 @@ extension SiteSettingsViewController {
         navigationController?.pushViewController(speedUpSiteSettingsViewController, animated: true)
     }
 
-    // MARK: AMP footer
+    // MARK: Footers
 
-    open override func tableView(_ tableView: UITableView, viewForFooterInSection section: Int) -> UIView? {
-        guard isTrafficSettingsSection(section) else {
-            return nil
-        }
-
-        let footer = UITableViewHeaderFooterView()
+    @objc(getTrafficSettingsSectionFooterView)
+    func trafficSettingsSectionFooterView() -> UIView {
+        let footer = commonFooterView()
         footer.textLabel?.text = NSLocalizedString("Your WordPress.com site supports the use of Accelerated Mobile Pages, a Google-led initiative that dramatically speeds up loading times on mobile devices.",
                                                    comment: "Footer for AMP Traffic Site Setting, should match Calypso.")
-        footer.textLabel?.numberOfLines = 0
-        footer.textLabel?.font = UIFont.preferredFont(forTextStyle: .footnote)
         footer.textLabel?.isUserInteractionEnabled = true
 
         let tap = UITapGestureRecognizer(target: self, action: #selector(handleAMPFooterTap(_:)))
         footer.addGestureRecognizer(tap)
         return footer
     }
 
+    @objc(getEditorSettingsSectionFooterView)
+    func editorSettingsSectionFooterView() -> UIView {
+        let footer = commonFooterView()
+        footer.textLabel?.text = NSLocalizedString("Edit new posts and pages with the block editor.", comment: "Explanation for the option to enable the block editor")
+        return footer
+    }
+
+    private func commonFooterView() -> UITableViewHeaderFooterView {
```

Comment:
Nitpicky, but when I read `commonFooterView` I imagine a view shared for all footers. Considering the usage, I'd suggest naming the method `makeFooterView` ([API Guidelines](https://swift.org/documentation/api-design-guidelines/#begin-factory-name-with-make)), or have a `configure*` method like we do with `WPStyleGuide`

Reply:
There's a `WPStyleGuide.configureTableViewSectionFooter` that I preferred not to modify since it's widely used.
`makeFooterView` songs great! 👍 
A local `configure*` sounds good too but a `make*` will save the extra init call. Just a small detail I guess? :)

I'll push the name change.